conf/app.routes

```diff
@@ -2,23 +2,23 @@
-GET         /no-agency-found                      uk.gov.hmrc.agentsubscriptionfrontend.controllers.CheckAgencyController.showNoAgencyFound
-GET         /confirm-your-agency                  uk.gov.hmrc.agentsubscriptionfrontend.controllers.CheckAgencyController.showConfirmYourAgency
-POST        /confirm-your-agency                  uk.gov.hmrc.agentsubscriptionfrontend.controllers.CheckAgencyController.submitConfirmYourAgency
-GET         /already-subscribed                   uk.gov.hmrc.agentsubscriptionfrontend.controllers.CheckAgencyController.showAlreadySubscribed
-GET         /has-other-enrolments                 uk.gov.hmrc.agentsubscriptionfrontend.controllers.CheckAgencyController.showHasOtherEnrolments
-
-GET         /enter-agent-code                     uk.gov.hmrc.agentsubscriptionfrontend.controllers.CheckAgencyController.invasiveCheckStart
-POST        /enter-agent-code                     uk.gov.hmrc.agentsubscriptionfrontend.controllers.CheckAgencyController.invasiveSaAgentCodePost
-GET         /enter-client-id                      uk.gov.hmrc.agentsubscriptionfrontend.controllers.CheckAgencyController.invasiveTaxPayerOptionGet
-POST        /enter-client-id                      uk.gov.hmrc.agentsubscriptionfrontend.controllers.CheckAgencyController.invasiveTaxPayerOption
+GET         /business-type                        uk.gov.hmrc.agentsubscriptionfrontend.controllers.BusinessIdentificationController.showBusinessTypeForm
+POST        /business-type                        uk.gov.hmrc.agentsubscriptionfrontend.controllers.BusinessIdentificationController.submitBusinessTypeForm
+GET         /business-details                     uk.gov.hmrc.agentsubscriptionfrontend.controllers.BusinessIdentificationController.showBusinessDetailsForm(businessType: Option[String])
+POST        /business-details                     uk.gov.hmrc.agentsubscriptionfrontend.controllers.BusinessIdentificationController.submitBusinessDetailsForm(businessType: Option[String])
+GET         /no-match                             uk.gov.hmrc.agentsubscriptionfrontend.controllers.BusinessIdentificationController.showNoAgencyFound
+GET         /confirm-business                     uk.gov.hmrc.agentsubscriptionfrontend.controllers.BusinessIdentificationController.showConfirmBusinessForm
+POST        /confirm-business                     uk.gov.hmrc.agentsubscriptionfrontend.controllers.BusinessIdentificationController.submitConfirmBusinessForm
+GET         /already-subscribed                   uk.gov.hmrc.agentsubscriptionfrontend.controllers.BusinessIdentificationController.showAlreadySubscribed
+GET         /create-new-account                   uk.gov.hmrc.agentsubscriptionfrontend.controllers.BusinessIdentificationController.showCreateNewAccount
+
+GET         /enter-agent-code                     uk.gov.hmrc.agentsubscriptionfrontend.controllers.BusinessIdentificationController.invasiveCheckStart
```

Comment:
Could consider aligning the controller action with the page name, for both `invasiveCheckStart` and `invasiveSaAgentCodePost`